pages/home/auth-providers/oauth2.mdx

```diff
@@ -65,7 +65,7 @@ HOOLI_CLIENT_SECRET="<your client secret>"
 
 Edit the `engine.yaml` file and add a new item to the `auth.providers` section:
 
-```yaml file=<rootDir>/examples/code/integrations/oauth2/config_provider_slim.engine.yaml
+```yaml file=<rootDir>/examples/code/home/configuration/engine/full_config.yaml#L8-L25 {11-18}
```

Comment:
By re-using the `full-config.yaml` file here, we guarantee this config example is always correct (thanks to CI validation)